packages/suite/src/utils/wallet/reviewTransactionUtils.ts

```diff
@@ -114,34 +114,31 @@ const constructOldFlow = ({
         precomposedTx.outputs.forEach(o => {
             // iterate only through "external" outputs (change output has addressParameters field instead of address)
             if ('address' in o) {
-                const tokenBundle = getCardanoTokenBundle(account, o)?.[0]; // send form supports one token per output
+                // const tokenBundle = getCardanoTokenBundle(account, o)?.[0]; // send form supports one token per output
 
                 // each output will include certain amount of ADA (cardano token outputs require ADA)
                 outputs.push({
                     type: 'regular_legacy',
-                    label: o.address,
-                    value: o.amount,
+                    value: o.address,
                 });
 
                 // if the output also includes a token then we need to render another row with the token
-                if (tokenBundle) {
-                    outputs.push({
-                        type: 'regular_legacy',
-                        label: o.address,
-                        value: tokenBundle.balance ?? '0',
-                        token: tokenBundle,
-                    });
-                }
+                // if (tokenBundle) {
```

Comment:
why commented out?

Reply:
Because Cardano flow is broken (and for Cardano tokens it's really broken) I came up with this "fix" because you were confirming the recipient address twice which with device display looks more weird.

But forgot to ask on @Hermez-cz opinion, I did this on my own.

Reply:
Since Cardano will not be in this PR I will bring it back as it is